pkg/utils/git/git.go

```diff
@@ -0,0 +1,55 @@
+package git
+
+import (
+	"fmt"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/config"
+)
+
+func RepoUrlFromDir(inputDir string) (string, error) {
+	_, remoteConfig, err := RepoFromDir(inputDir)
+	if err != nil {
+		return "", err
+	}
+	repoRemoteUrl := remoteConfig.URLs[0]
```

Comment:
Should we return all and display a user selection if it's not clear? Should we maybe take origin as default if there's more than one remote repository?

Use case:
- contribution to repo without write rights
- original repo is "upstream"
- fork is "origin"
Most likely the ignore should be in origin, not upstream, as we are not commiter in upstream.

Reply:
I think we anyways have some duplication [here](https://github.com/snyk/code-client-go/blob/30592824b40dc41d7a720b2f27b2581fdc3e7808/internal/util/repository.go#L25). IMHO a great candidate to deduplicate when both repos are merged.